.github/workflows/cats.yml

```diff
@@ -0,0 +1,14 @@
+name: Cats 😺
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  thank-you:
+    name: A cat for your effort!
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ruairidhwm/action-cats@1.0.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.github_token }}
```

Comment:
Not sure that we allowed to give GH token to 3rd party action :(

Reply:
Will check with Sec. TY!